cli/crypto/mod_test.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"bytes"
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.dedis.ch/dela/cli"
+)
+
+func TestMain(t *testing.T) {
+	builder = &fakeBuilder{}
+	main()
```

Comment:
what is the point of this code?
No asserts are being made about it and it seems irrelevant to the rest of the test.

Reply:
if the idea is to test the 'happy path' , consider splitting it out in a separate test asserting that nothing is sent on the printer.